src/wasm.h

```diff
@@ -1427,15 +1427,15 @@ class Function {
   }
 };
 
+enum class Kind {
```

Comment:
Yeah, I want to call it Global too, but given Global means a global variable, it's too confusing. We'd need to call Global "Var" or something like that, diverging from other repos' conventions.

ModuleValue is a possibility, but I'm not sure. It's not familiar like Global, and things might change (maybe functions aren't just owned by modules some day?).

ExternalKind seems best, it's consistent with the design and wabt repos, and the main downside is we might need to add an InternalKind some day, which isn't the end of the world even if it comes to that.